tests/test_pygwalker.py

```diff
@@ -62,3 +69,18 @@ def test_process_parameter_change_with_appearance(data):
     result = gwalker._process_param_change(params)
     assert result["appearance"]==appearance
 
+@pytest.mark.xfail(reason="I did not know how to fix this. When I did the app_demo.py example would fail.""")
+def test_change_appearance_forth_and_back(data, default_appearance):
```

Comment:
I hope you can get this working? It can wait though as dynamically changing the appearance it not something that will be done a lot.

Reply:
It should be fixed together with a releated problem. If the appearance is reset to `"panel"` the `app_demo.py` will error. The cause is that `"panel"` is sent to client side and applied. I could not figure out how to fix this.

I tried to fix it in `_process_param_change`. But when I tried I got other exceptions. Its like its not allow with try/ except in that function and that if there are errors then they will not be logged but some bokeh error will.

Reply:
Fixed this by having only the 3 appearances supported by Graphic Walker. Is simpler. It actually also follows the concept I've proposed here https://github.com/holoviz/panel/pull/7406.